homeassistant/components/roborock/models.py

```diff
@@ -0,0 +1,12 @@
+"""Roborock Models."""
+from dataclasses import dataclass
+
+from roborock import HomeDataProduct, RoborockDeviceProp, RoborockLocalDeviceInfo
+
+
+@dataclass
+class RoborockHassDeviceInfo(RoborockLocalDeviceInfo):
```

Comment:
Optional: Preferring composition to inheritance would probably improve readability  since this is subclassing a dataclass in another package